.github/workflows/prepare-release.yml

```diff
@@ -8,7 +8,7 @@ on:
         required: true
 
 jobs:
-  prepare-relese:
+  prepare-release:
```

Comment:
I've no idea if changing the job name will break anything, so please let me know if this needs to be reverted.

Reply:
Doesn't break anything, no worries :)